azure-test/tests/azure_storage_blob/test-list-query.sql

```diff
@@ -1,3 +1,3 @@
 select name, storage_account_name, container_name, type, is_snapshot, access_tier, deleted, server_encrypted, content_length, lease_state, region, resource_group, subscription_id
 from azure.azure_storage_blob
-where title = '{{ resourceName }}'
\ No newline at end of file
+where resource_group = '{{ resourceName }}' and storage_account_name = '{{ resourceName }}' and region = 'eastus' and name = '{{ resourceName }}';
```

Comment:
Can we add reference instead of fixed value `eastus`?
```suggestion
where resource_group = '{{ resourceName }}' and storage_account_name = '{{ resourceName }}' and region = 'eastus' and name = '{{ resourceName }}';
```